lib/Parse/Lexer.cpp

```diff
@@ -716,6 +722,12 @@ static bool isRightBound(const char *tokEnd, bool isLeftBound,
     else
       return true;
 
+  case '\xC2':
+    if (tokEnd[1] == '\xA0')
```

Comment:
Can you confirm that this dereference is safe? Can we assume that the buffer is always null-terminated?

Reply:
Yes, it's guaranteed. https://github.com/apple/swift/blob/9ba05efb/lib/Parse/Lexer.cpp#L188